vendor/deno-dom/element.ts

```diff
@@ -370,4 +370,30 @@ export class Element extends Node {
     }
     return null
   }
+
+  insertBefore(newNode: Node, existingNode: Node | null) {
```

Comment:
Not ready for merge yet, `insertBefore` is a `Node` method so I will move it shortly.